.github/workflows/release_jars.yml

```diff
@@ -0,0 +1,48 @@
+# This action will publish jars that can be used by third parties.
+# Pretty much everything is done through gradle.
+# The version used will be according to the axion-release-plugin, meaning it will take a tag if present,
+# if not it will create a SNAPSHOT version.
+# gradle will compile, sign then upload artifacts to a Sonatype staging repo.
+# See https://s01.oss.sonatype.org for accessing these repos.
+# At this point it should manually be closed, which will trigger acceptance tests for maven central (but not transfer yet)
+# Once closed, thew repo is available for testing.
+# After testing, it can be manually promoted on the sonatype site, which will then publish to maven central.
+name: Upload Release Jars to Sonatype
```

Comment:
What do you think if we  add the jobs implemented here to https://github.com/MobilityData/gtfs-validator/blob/85beb70856203dd8edc7f4b06ef5b89a840e5908/.github/workflows/publish_assets.yml#L1 to avoid sync issues between what we have in the Github repository and what is published in maven

Reply:
So you want to combine the two actions?
Or have one trigger the other

Reply:
Combining the two actions seems to be a reasonable solution.

Reply:
Combining is possible, but having a tag push as a trigger instead of a release event will not work since publish_release uploads to a release and needs the release event to obtain the release url.

Reply:
I see your point. My bad, I was looking at the default example from `upload-release-asset` action and the implementation is the other way around. They create a release as part of the tag push: https://github.com/actions/upload-release-asset#example-workflow---upload-a-release-asset. 
To conclude, let's leave the pre-release and release triggers.

Reply:
Still kept them combined but added a workflow_dispatched trigger that is used only by the sonatype upload.